src/main/java/com/adaptavist/tm4j/jenkins/http/Tm4jJiraRestClient.java

```diff
@@ -24,40 +25,40 @@
 
 public class Tm4jJiraRestClient {
 
-    private final JiraInstance jiraInstance;
+    private final Instance jiraInstance;
 
-    public Tm4jJiraRestClient(List<JiraInstance> jiraInstances, String serverAddress) throws Exception {
+    public Tm4jJiraRestClient(List<Instance> jiraInstances, String serverAddress) throws Exception {
         jiraInstance = getTm4jInstance(jiraInstances, serverAddress);
     }
 
     public void uploadCucumberFile(String directory, String filePath, String projectKey, Boolean autoCreateTestCases, final PrintStream logger) throws Exception {
         File file = new FileReader().getZip(directory, filePath);
-        HttpResponse<JsonNode> jsonResponse = jiraInstance.importCucumberBuildResult(projectKey, autoCreateTestCases, file);
-        processImportingResultsResponse(jsonResponse, logger);
-        if(!file.delete()) {
+        HttpResponse<JsonNode> jsonResponse = jiraInstance.exportCucumberFormatBuildResult(projectKey, autoCreateTestCases, file);
+        processExportingResultsResponse(jsonResponse, logger);
+        if (!file.delete()) {
             logger.printf("%s The generated ZIP file couldn't be deleted. Please check folder permissions and delete the file manually: " + file.getAbsolutePath() + " %n", INFO);
         }
     }
 
     public void uploadCustomFormatFile(String directory, String filePath, String projectKey, Boolean autoCreateTestCases, final PrintStream logger) throws Exception {
         File file = new FileReader().getZip(directory, filePath);
-        HttpResponse<JsonNode> jsonResponse = jiraInstance.importCustomFormatBuildResult(projectKey, autoCreateTestCases, file);
-        processImportingResultsResponse(jsonResponse, logger);
-        if(!file.delete()) {
+        HttpResponse<JsonNode> jsonResponse = jiraInstance.exportCustomFormatBuildResult(projectKey, autoCreateTestCases, file);
```

Comment:
In this case i'm confused, `jiraInstance` is importing from or exporting from?

Reply:
Usually the concept is `import from` and `export to` - so the plugin exports the custom/cucumber results and imports the feature files. I think I've renamed everything in the plugin based on this assumption which, imho, makes more sense when reading the code.

Reply:
As per my previous comments, what if we stick to the naming convention we have for the build steps title? "publishCustomFormatBuildResult". I think that would be clearer.

Reply:
That's the point @dragosmc. `tm4jJiraRestClient.exportCustomFormatBuildResult ` would make sense for me but `jiraInstance.exportCustomFormatBuildResult` doesn't make sense because jira doesn't export anything, it imports. Anyway, free interpretation here :)